Frontend/src/pages/registration_administration/components/RegistrationAdministrationList.jsx

```diff
@@ -122,6 +123,19 @@ export default function RegistrationAdministrationList() {
     },
   })
 
+  const { data: PIIData } = useQuery({
```

Comment:
```suggestion
  const { data: piiData } = useQuery({
```
Only components should be capitalized.

The loading message should probably be shown if either this or registrations is loading. Or maybe it's fine ignore this loading state since dob/email aren't visible with the initial default column visibility settings?

Reply:
yes, that's why I didn't add a loading indicator